### ISSUE TITLE
TimeDistributed(Dropout()) with the same dropout mask

### ISSUE THREAD

**Issue by bzhong2** [tensorflow/tensorflow#39631]
**System information**
- TensorFlow version (you are using): 1.14
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.**

Here is an example block of my code. I am trying to apply a time distributed dropout to the output of a many to many GRU. I would like to keep the dropout to have the same dropout mask for all time steps. However, I did not find a solution to this purpose based on the current API. Did I miss anything or is it a new feature on the roadmap? Thanks a lot! 

```
from tensorflow.keras.layers import Dense, Input, GRU, Dropout, TimeDistributed
x= TimeDistributed(Dense(512, activation='relu', kernel_regularizer=l2(1e-5), \
                bias_regularizer=l2(1e-5), name='cam_fc'))(input_tensor)
out = GRU(
                512,
                dropout=0.1,
                recurrent_dropout=0.1,
                activation='relu', 
                kernel_regularizer=l2(1e-5),
                bias_regularizer=l2(1e-5),
                return_sequences=True, 
                name='intentNet_gru')(x, training=self.is_train)

out = TimeDistributed(Dropout(0.1))(out, training=self.is_train)
```

**Comment by qlzh727** [comment]
I think what TimeDistributed does is that it slide the inputs based on timestep, and feed each timestep one by one to the wrapped layer. Note that when feeding multiple input to a dropout layer, each of them will get different dropout mask, which is aligned with current behavior.

If you want to apply same mask across time steps, what you can do is just using one dropout layer, but with noise_shape param specified. See https://www.tensorflow.org/api_docs/python/tf/keras/layers/Dropout for more details.
